app/helpers/generic_phrases_helper.rb

```diff
@@ -28,57 +28,48 @@ def generic_phrase_table_headers(presenter)
     column_map.values
   end
 
-  def map_verbs_to_table(presenter)
+  def create_options_for_select_list(items, selected)
     options = [
       {
         text: "Select...",
         value: "",
       },
     ]
 
-    options += presenter.verbs.map do |verb|
+    options += items.map do |item|
       {
-        text: verb,
-        value: verb,
-        selected: verb == presenter.verb,
+        text: item,
+        value: item,
+        selected: item == selected,
       }
     end
 
     options
   end
 
-  def map_adjectives_to_table(presenter)
-    options = [
-      {
-        text: "Select...",
-        value: "",
-      },
-    ]
-
-    options += presenter.adjectives.map do |adj|
-      {
-        text: adj,
-        value: adj,
-        selected: adj == presenter.adjective,
-      }
+  def map_most_frequent_exact_matches_to_table(presenter)
```

Comment:
As I said in some other review, I think things like this belong in the presenter but we will sort that out at some other time

Reply:
To be fair, more of it is in the presenter now than it was before. I think too that `create_options_for_select_list` should belong in a helper as it's pretty generic.